### PR TITLE
Add general usage to readme

### PR DESCRIPTION
**Pull request by notlee**
Links to instructions on how to include Origami components. https://github.com/Financial-Times/origami/issues/59

**This was added by a script.** If approved I'll run the script to apply the update to other components and automatically tag a new patch release. I'll run against all non-utility, origami owned components. I'll follow up with a PR for non-origami owned components.

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-buttons bundle size difference from 6.0.16 to 60e8cc03f38b8efbd34f84457f68675f6ddd4743
No bundle size differences found.

File: README.md [hunk 1]
```diff
@@ -2,13 +2,19 @@
...
 - [Sass](#sass)
 - [References](#references)
 - [Migration](#migration)
 - [Contact](#contact)
 - [Licence](#licence)
 
+## Usage
+
+There are two ways to include `o-buttons` in your project, consistent with other Origami components. See [how
+to include Origami components in your project](https://origami.ft.com/docs/components/#including-origami-components-in-your-project).
```
Comment (chee) [inline README.md:16]: I think the first sentence might need another draft, it feels a little gummy. I'm not sure how to improve it though. Maybe remove it, be more conversational?

```suggestion
Check out [how to include Origami components in your project](https://origami.ft.com/docs/components/#including-origami-components-in-your-project).
```

Comment (notlee) [reply README.md:16]: Hmm it is gummy. I feel your suggestion is better but maybe a little abrupt, might if I append "to get started with `o-buttons`"?

```suggestion
Check out [how to include Origami components in your project](https://origami.ft.com/docs/components/#including-origami-components-in-your-project) to get started with `o-buttons`.
```

Comment (chee) [reply README.md:16]: ooh that's good i think, i like that a lot

**Comment by origamiserviceuser** [comment]
o-buttons bundle size difference from 6.0.16 to 2a36b6e0ca7b1401bc08bd9deb5a1e69998522dc
No bundle size differences found.

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v6.0.17 :tada:

**Pull request merged**